test/soca/create_bkgdb.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import os
+import shutil
+from solo.configuration import Configuration
+from solo.nice_dict import NiceDict
+import ufsda.r2d2
+
+if __name__ == "__main__":
+
+    # Setup the shared R2D2 databases
+    ufsda.r2d2.setup(r2d2_config_yaml='r2d2_config.yaml', shared_root='./r2d2-shared')
+
+    # Invent backgrounds
+    # MOM6 format: MOM.res.2019-04-15-01-00-00.nc MOM.res.2019-04-15-02-00-00.nc  MOM.res.2019-04-15-03-00-00.nc ...
+    # CICE6 format: iced.2019-04-15-00000.nc iced.2019-04-15-03600.nc  iced.2019-04-15-07200.nc ...
+
+    bkgdir = os.getenv('TESTDATA')
+    shutil.copyfile(os.path.join(bkgdir, 'MOM.res.nc'), './RESTART/MOM.res.2018-04-15-09-00-00.nc', follow_symlinks=True)
```

Comment:
create a datetime object, loop with an interval of 3 hours instead of writing out 4 nearly duplicate lines?

Reply:
sure, might save a line or 2 ...

Reply:
Probably not but cleaner :-) up to you to change if you'd like